.coafile

```diff
@@ -22,3 +22,8 @@ bears = MarkdownBear, AlexBear
 
 [Git]
 bears = GitCommitBear
+
+[YAML]
+files = **/*.yml
+bears = YAMLLintBear
+ignore = node_modules/**/*.yml
```

Comment:
You can just ignore `node_modules/**` , as all files under that are undesirable

Reply:
done :+1: